image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Image provides access to internal image info
```

Comment:
@Random-Liu thanks for the explanation.

> Containerd image store is indexed by image reference (docker.io/library/busybox:latest),...however, in cri-containerd we need to index image by image ID

This doesn't strike me as an uncommon requirement. Perhaps there is a containerd enhancement to be made here to allow indexing by image id? (If that isn't already possible, I'm not sure, @stevvooe ?)